src/codegen/generator.rs

```diff
@@ -55,14 +55,14 @@ pub fn exec_cmd(name: &str, path: &Path, cmd: &str, args: &[&str]) {
     }
 }
 
-/// A type that derives the contents of a data package from a `DataPackage` descriptor.
+/// A type that derives the contents of a data package from a `Dataset`.
 pub trait Generator {
     /// The data package that the generator is processing.
```

Comment:
Doesn't the docstring also need to change?
```suggestion
    /// The dataset that the generator is processing.
```

Reply:
yes!

Reply:
Fixed in next-upstack PR.